### PR TITLE
Update enterprise dropdown

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

Update enterprise dropdown with an array of updates

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/#enterprise
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1YBdQvLuqEpEQr_QqyycxMhZr4OaLapMlNJyYKCmPJsQ/edit#
)

### CODE REVIEW

File: templates/templates/_navigation-enterprise-h.html [hunk 1]
```diff
@@ -144,9 +144,9 @@ <h4 class="p-muted-heading"><a href="/server">Data centre&nbsp;&rsaquo;</a></h4>
         <h4 class="p-muted-heading"><a href="/kubeflow">AI &amp; ML&nbsp;&rsaquo;</a></h4>
         <ul class="p-text-list--small is-bordered">
           <li class="p-list__item"><a href="/kubernetes/features">Kubernetes support for AI/ML</a></li>
-          <li class="p-list__item"><a href="/blog/ai-ml-model-ubuntu">Deploy your first model with Kubeflow</a></li>
+          <li class="p-list__item"><a href="/kubeflow/install">Deploy your first model with Kubeflow</a></li>
```
Comment (anthonydillon) [inline templates/templates/_navigation-enterprise-h.html]: This page doesn't result in a deployed model but is a much better start to kubeflow. Maybe replace the text to something like "Start using kubeflow with MicroK8s".

Comment (pmahnke) [reply templates/templates/_navigation-enterprise-h.html]: ok

**Comment by pmahnke** [comment]
@anthonydillon ok, ok, I did a _much_ bigger clean-up, changed a lot, added a lot.  I think it is a lot more up-to-date and better... happy for suggestions.

**Comment by pmahnke** [comment]
@anthonydillon not sure we wanted to change it there? @matthewpaulthomas thoughts?

**Pull request merged**